tests/test_utils.py

```diff
@@ -196,6 +262,63 @@ def test_sub_pixel_read(_source_image):
     sub_pixel_read(test_image, pillow_test_image, bounds, ow, oh)
 
 
+def test_aligned_padded_sub_pixel_read():
+    """Test sub-pixel numpy image reads with pixel-aligned bounds."""
+    image_path = Path(__file__).parent / "data" / "source_image.png"
```

Comment:
I think we should replace this with _source_image as the files have been moved from `data` to tiaserver.

Reply:
Fixed in this commit https://github.com/TIA-Lab/tiatoolbox/pull/61/commits/8b687c59ac3e9b5d2fc296ded0cae17e148670d3